docs/ml-collaborative-filtering.md

```diff
@@ -60,7 +60,7 @@ best parameter learned from a sampled subset to the full dataset and expect simi
 <div class="codetabs">
 <div data-lang="scala" markdown="1">
 
-In the following example, we load rating data from the
+In the following example, we load ratings data from the
```

Comment:
Although I think "rating data" is actually OK, so is "ratings data". I think the whole change LGTM